Sources/ActionSheet/ActionSheetSortSectionHeader.swift

```diff
@@ -123,6 +123,7 @@ class ActionSheetSortSectionHeader: ActionSheetSectionHeader {
     @objc private func updateTheme() {
         backgroundColor = PresentationTheme.current.colors.background
         descendingLabel.textColor = PresentationTheme.current.colors.cellTextColor
+        gridLayoutLabel.textColor = PresentationTheme.current.colors.cellTextColor
```

Comment:
`GRID_LAYOUT` didn't update text color for new theme. There is no `GRID_LAYOUT` option in production - so I failed to make screenshot with this bug.